src/Microsoft.ML.Data/DataView/RowToRowMapperTransform.cs

```diff
@@ -245,11 +237,14 @@ void ISaveAsPfa.SaveAsPfa(BoundPfaContext ctx)
             }
         }
 
-        public Func<int, bool> GetDependencies(Func<int, bool> predicate)
+        /// <summary>
+        /// Given a set of output columns, return the input columns that are needed to generate those output columns.
+        /// </summary>
+        IEnumerable<Schema.Column> IRowToRowMapper.GetDependencies(IEnumerable<Schema.Column> dependingColumns)
         {
-            Func<int, bool> predicateInput;
-            GetActive(predicate, out predicateInput);
-            return predicateInput;
+            var predicate = RowCursorUtils.FromColumnsToPredicate(dependingColumns, OutputSchema);
+            GetActive(predicate, out IEnumerable<Schema.Column> inputColumns);
```

Comment:
>IEnumerable<Schema.Column> [](start = 37, length = 26)

Don't be afraid of `out var`. This will ultimately make @eerhardt's job of renaming this sort of thing easier if you want a selfless reason to do so.